external/java/README.md

```diff
@@ -0,0 +1,45 @@
+# Publishing to Maven
+
+Get an account on Sonatype JIRA: https://issues.sonatype.org/secure/Signup!default.jspa
+
+Tell your username to Jeff, who will file a ticket to add you
+to have access to publish.
```

Comment:
I don't really understand this part. This instruction is to help other people to publish their own version of cothority-java? If we need our library to be on the maven repo, it's easier to just publish it directly by ourselves.

Reply:
It is to tell you how to publish. But maybe it does not belong in the public repo.

Reply:
OK, I have moved these notes into the "Operations Notes" file in the Google Drive.